content/articles/how-to-make-a-discord-bot-using-python/index.md

```diff
@@ -0,0 +1,292 @@
+
+Now that the bot is finally ready to go, where will it go?
+
+ If a bot user isn't communicating with other users, it's useless. So you'll then create a guild for your bot so that it may communicate with other people.
+
+4.**Forming a Guild (a Server)**
+
+A guild (as it's referred to in Discord's user interface) is a collection of routes where members may communicate.
+
+**Notes: While the terms guild and server are equivalent, we will use the term "guild" in this article because the APIs use the exact phrase. We will only use the term "server" in the graphical user interface to refer to a guild.**
+
+Now, go to your Discord home (link) page and establish a guild:
+
+You may see and connect with friends, direct messages, and guilds through this home page. To Add a Server, click the "+" button from the left side panel of the web page:
+
+![AddFriend](./img7.png)
+
+On the upcoming screen, select "Create a server" and give the server a name. Then, you can interact with yourself or request some people to communicate with you after the server is up and running. We'll be inviting our bot to communicate with us soon. 
+
+You'll be able to view the users on the right side and the channels on the left panel when you've done building your guild:
+
+![screen](./img8.png)
+
+Further, the final stage is to register the bot with the new guild at Discord.
+
+### Create a discord bot using discord.py library
+
+The bot's programming will be written in Python using the discord.py package. discord.py is a Discord API wrapper that makes creating a Discord bot in Python much more effortless.
+
+
+#### **Installing** `discord.py`  and creating a Repl
+
+Any coding editor may be used to create the bot on your PC. However, we'll utilize Repl.it in this tutorial as it makes things easier for anybody to follow along. Repl.it is a web-based IDE.
+
+Begin by visiting Repl.it. Make a new Repl.it and set the language to "Python."
+
+Simply put, import discord at the start of main.py to utilize the discord.py module. Then, when you hit the "run" button on Repl.it, it will automatically install this dependency.
+
+On macOS, run this command to install discord.py if you wish to develop the bot locally:
+
+```python3 -m pip install -U discord.py```
+
+You may have to use pip3 instead of pip.
+
+```py -3 -m pip install -U discord.py```
+
+#### Setting up Discord event for your bot
+
+The structure of events is central to `discord.py`. An event is anything to which you pay attention and then respond. So, for example, when a message arrives, you will be notified via an event to which you can reply.
+
+Let's build a bot that responds to a given phrase. This simple bot code is derived from the discord.py manual, as is the [documentation](https://discordpy.readthedocs.io/en/latest/quickstart.html#a-minimal-bot). Later on, we'll add more functionality to the bot.
+
+This code should be added to main.py. (You may rename the file to whatever you like, don't call it discord.py.)
```

Comment:
Explaining why one shouldn't name it discord.py would definitely help. This is a problem many people encounter regularly and getting this knowledge in this article might help them elsewhere.